src/modules/cmdpal/Microsoft.CmdPal.UI/Controls/FormControl.xaml.cs

```diff
@@ -0,0 +1,86 @@
+// Copyright (c) Microsoft Corporation
+// The Microsoft Corporation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using AdaptiveCards.ObjectModel.WinUI3;
+using AdaptiveCards.Rendering.WinUI3;
+using Microsoft.CmdPal.UI.ViewModels;
+using Microsoft.UI.Xaml.Controls;
+using Windows.UI.ViewManagement;
+
+namespace Microsoft.CmdPal.UI.Controls;
+
+public sealed partial class FormControl : UserControl
+{
+    private static readonly AdaptiveCardRenderer _renderer;
+    private FormViewModel? _viewModel;
+
+    public FormViewModel? ViewModel { get => _viewModel; set => AttachViewModel(value); }
+
+    static FormControl()
+    {
+        // yep this is the way to check if you're in light theme or dark.
+        // yep it's this dumb
+        var settings = new UISettings();
+        var foreground = settings.GetColorValue(UIColorType.Foreground);
+        var lightTheme = foreground.R < 128;
```

Comment:
Can we use `ActualTheme` prop here? https://learn.microsoft.com/windows/windows-app-sdk/api/winrt/microsoft.ui.xaml.frameworkelement.actualtheme (there's an event to listen to changes too)